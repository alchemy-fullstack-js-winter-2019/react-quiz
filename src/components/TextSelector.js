import React from 'react';
import PropTypes from 'prop-types';


function TextSelector({ handleChange }) {

    return (
        <label>IMAGE TEXT 
            <input type='text' name="ImageText" onChange={handleChange}/>
        </label>
    );
}

TextSelector.propTypes = {
    handleChange: PropTypes.func.isRequired
};

export default TextSelector;
