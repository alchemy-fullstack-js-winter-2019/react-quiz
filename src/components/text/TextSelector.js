import React, { Fragment } from 'react';

function TextSelector() {
  return (
    <Fragment>
      <h2>I am text</h2>
    </Fragment>
  );
}

export default TextSelector;
