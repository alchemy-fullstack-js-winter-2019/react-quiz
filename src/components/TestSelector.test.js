import React from 'react';
import renderer from 'react-test-renderer';
import TextSelector from './TextSelector';

describe('Color', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <TextSelector text="hello" />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

