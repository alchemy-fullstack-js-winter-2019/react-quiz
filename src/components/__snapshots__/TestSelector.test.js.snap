// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color matches a snapshot 1`] = `
Array [
  <h1>
     Text 
  </h1>,
  <input
    name="text"
    placeholder="text"
    text="hello"
    type="text"
  />,
]
`;
